pkgs/development/tools/earthly/default.nix

```diff
@@ -13,6 +13,18 @@ buildGoModule rec {
 
   vendorSha256 = "sha256-q3dDV0eop2NxXHFrlppWsZrO2Hz1q5xhs1DnB6PvG9g=";
 
+  buildFlagsArray = ''
+    -ldflags=
```

Comment:
```suggestion
    -ldflags=
      -s -w
```

Reply:
what do they do?

Reply:
ah shrinking
https://lukeeckley.com/post/useful-go-build-flags/

`We can omit the symbol table, debug information and the DWARF table with the following flags: -s -w`